### PR TITLE
Added a page regarding icons.

### PR DESCRIPTION
**Pull request by makerimages**
This is missing from the docs. Added it.

### CODE REVIEW

**Comment by makerimages** [comment]
Sorry, messed up the gemfile by accident. Fixed now.

**Comment by makerimages** [comment]
That's a squash of the previous commits. Message not right. Sorry.

**Comment by makerimages** [comment]
Anything?

/cc @izuzak

File: book/03-hacking-atom/sections/06-iconography.asc [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  icon-something
+```
+
+And so, to get the same icon to appear as a span, you could do this:
+
+```coffeescript
+  @span class: 'icon-device-desktop'
+```
+
+The result is the same: the  https://octicons.github.com/icon/device-desktop/[device-desktop] icon appears.
```
Comment (50Wliu) [inline book/03-hacking-atom/sections/06-iconography.asc]: There's two spaces between `the` and `https:`

File: book/03-hacking-atom/sections/06-iconography.asc [hunk 2]
```diff
@@ -0,0 +1,27 @@
+=== Iconography
+
+Every so often you come to a point in time, where, to represent something, an icon is better off than words are.
+Since Atom is made by GitHub, their https://octicons.github.com/[Octicons] font is available for use.
```
Comment (simurai) [inline book/03-hacking-atom/sections/06-iconography.asc]: > icon is better off than words

Some argue that "[The best icon is a text label](http://thomasbyttebier.be/blog/the-best-icon-is-a-text-label)". So maybe we should encourage people to use text labels if possible and only if space gets tight fall back to an icon. Or often icons get used in addition to a text label.

> Since Atom is made by GitHub

Not sure about mentioning this as the main reason. I guess it is, but just sounds a bit strange in this context. How about that whole first paragraph simplified to something like:

> Atom comes bundled with the [Octicons](https://octicons.github.com/). You can use these icons in your packages if space for a text label is insufficient or to add a visual clue to an existing label.

File: book/03-hacking-atom/sections/06-iconography.asc [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+===== Usage
+
+Atom's usage of https://octicons.github.com/[Octicons] differs just a bit from the standard usage of them, that you'd find, say, in a web project.
```
Comment (simurai) [inline book/03-hacking-atom/sections/06-iconography.asc]: How about:

> Atom's usage of the Octicons differs just a bit from the [standard usage](https://octicons.github.com/usage/).

Since they're already linked above. Also mentioning "web project" feels not that necessary.

File: book/03-hacking-atom/sections/06-iconography.asc [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+The biggest difference is in the naming of the icon classes. In regular webdev, you'd create something like this:
+
+```html
+  <span class = "octicon octicon-device-desktop"></span>
```
Comment (simurai) [inline book/03-hacking-atom/sections/06-iconography.asc]: The space before and after the `=` can be removed.

**Comment by simurai** [comment]
Sorry, after adding a couple of inline comments it kinda became too complex. So I made another PR https://github.com/makerimages/docs/pull/1 on top of this one that would rewrite it a bit. :innocent:

:+1: on adding this section, especially since the usage differs from the standard Octicons.

File: book/A-upgrading/sections/ui-theme.asc [hunk 5]
```diff
@@ -68,8 +68,7 @@ atom-text-editor::shadow .highlight.my-linter {
 }
 ```
 
-Check out the https://github.com/atom/find-and-replace/blob/95351f261bc384960a69b66bf12eae8002da63f9/styles/find-and-replace.less#L10
-[find-and-replace]package for another example of using `::shadow` to pierce the shadow DOM.
+Check out the https://github.com/atom/find-and-replace/blob/95351f261bc384960a69b66bf12eae8002da63f9/stylesheets/find-and-replace.less#L9-L29[find-and-replace] package for another example of using `::shadow` to pierce the shadow DOM.
```
Comment (simurai) [inline book/A-upgrading/sections/ui-theme.asc]: This looks like it shouldn't be part of this PR. We can still keep it since it fixes that link.

Comment (makerimages) [inline book/A-upgrading/sections/ui-theme.asc]: Yeah, don't know what went wrong there :stuck_out_tongue:

**Comment by makerimages** [comment]
I fixed the link markup, that @simurai broke.

**Comment by simurai** [comment]
> I fixed the link markup, that @simurai broke.

:bow:

**Comment by lee-dohm** [comment]
Thanks @makerimages!
